Source/LibationFileManager/PictureStorage.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace LibationFileManager
 {
-	public enum PictureSize { _80x80 = 80, _300x300 = 300, _500x500 = 500 }
+	public enum PictureSize { _80x80 = 80, _300x300 = 300, _500x500 = 500 , _1215x1215 = 1215 }
```

Comment:
Per other conversation: as this code is here, this will apply the 1215 size to old/smaller/reduced-size image ID

Reply:
It only matters if you're requesting an image with that size.  I removed it for now because the cover should be the Large native size, not reduced to 1215.